Fechas y Calendario.

In [7]:
%%bash
whoami

root


In [3]:
%%bash
date

Tue Oct 13 17:38:59 CDT 2020


CDT: Central Day Light Time.
UTC: Universal Coordinated Time.

In [8]:
%%bash
cal 02 1911

   February 1911      
Su Mo Tu We Th Fr Sa  
          1  2  3  4  
 5  6  7  8  9 10 11  
12 13 14 15 16 17 18  
19 20 21 22 23 24 25  
26 27 28              
                      


El 25 de Febrero de 1911, Fecha de nacimiento de Carlos Graef cayó en sabado. 

In [11]:
%%bash
cal -j 07 1913

         July 1913           
 Su  Mo  Tu  We  Th  Fr  Sa  
        182 183 184 185 186  
187 188 189 190 191 192 193  
194 195 196 197 198 199 200  
201 202 203 204 205 206 207  
208 209 210 211 212          
                             


Alberto Barajas, nació el 17 de Julio de 1913, tal fecha cae en Jueves y el día es el 198 del año.

In [12]:
%%bash
uname

Linux


In [13]:
%%bash
uname -a

Linux Ignacio-PC 4.4.0-19041-Microsoft #488-Microsoft Mon Sep 01 13:43:00 PST 2020 x86_64 x86_64 x86_64 GNU/Linux


In [14]:
%%bash
uname -r

4.4.0-19041-Microsoft


Mi versión de Kernel es 4.4.0-19041-Microsoft, el segundo digito al ser par indica que es de producción.

Manejo e Información de Directorios y Archivos. 

In [11]:
pwd

'C:\\Users\\Ignacio\\Documents\\Pochas\\Computacion'

In [3]:
%%bash
mkdir susanaregalado

In [2]:
%%bash
cd /mnt/c/Users/Ignacio/Documents/Pochas/Computacion/susanaregalado

In [13]:
%%bash 
mkdir regalado

In [16]:
%%bash
mkdir espino

In [1]:
%%bash 
touch ola

In [4]:
%%bash 
touch .ola

El comando cd ~ nos manda a la raíz del sistema.

In [5]:
%%bash 
ls -R

.:
Computacion2020-8093-master.zip
Julia.jl
LINUX_ComandosBasicos.ipynb
LINUX_PrimerosComandos.ipynb
P1.ipynb
Practica 1, Regalado Espino Cinthya Susana.ipynb
Project.toml
README.md
Tarea 1 computación
apt.txt
requirements.txt
susanaregalado

./Tarea 1 computación:
Instalación de Bash y Julia en Jupyter Lab..jpeg
Tarea 1. Regalado Espino Cinthya Susana..ipynb

./susanaregalado:
espino
regalado

./susanaregalado/espino:
ola

./susanaregalado/regalado:


Lo que sucede con el archivo touch .ola es que se encuentra oculto en el directorio.

Para poder ver este archivo se necesita utilizar el comando ls -a. 